name-generator.py

```diff
@@ -0,0 +1,43 @@
+import random
+
+# Function to randomly select a specified number of names from a list of names
+def random_name_selector(names, num_selections):
+    # Check if the number of selections is valid (greater than 0 and less than or equal to the number of names)
+    if num_selections > 0 and num_selections <= len(names):
+        # Randomly select the specified number of names
+        selected_names = random.sample(names, num_selections)
+        return selected_names
+    else:
+        # Return an error message if the number of selections is invalid
+        return "Invalid input: number of selections should be between 1 and the length of the shortlist."
```

Comment:
Use f-strings for error message: The code currently returns an error message as a string. It would be more Pythonic to use f-strings to format the error message, like so: return f"Invalid input: number of selections should be between 1 and {len(names)}.".